databricks-sdk-java/src/main/java/com/databricks/sdk/core/DatabricksConfig.java

```diff
@@ -38,6 +38,15 @@ public class DatabricksConfig {
   @ConfigAttribute(env = "DATABRICKS_REDIRECT_URL", auth = "oauth")
   private String redirectUrl;
 
+  /**
+   * The OpenID Connect discovery URL used to retrieve OIDC configuration and endpoints.
+   *
+   * <p><b>Note:</b> This API is experimental and may change or be removed in future releases
+   * without notice.
+   */
+  @ConfigAttribute(env = "DISCOVERY_URL")
```

Comment:
We'll keep all environment variables prefixed with `DATABRICKS_` so they don't clash with other applications.
```suggestion
  @ConfigAttribute(env = "DATABRICKS_DISCOVERY_URL")
```